packages/react-core/src/components/BackToTop/__tests__/BackToTop.test.tsx

```diff
@@ -1,22 +1,93 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import { BackToTop } from '../BackToTop';
+import userEvent from '@testing-library/user-event';
 
```

Comment:
Can you add a basic rendering test here like you did for [Avatar](https://github.com/patternfly/patternfly-react/blob/eafbe0f0313de8c721a45e87ba182342ed9107aa/packages/react-core/src/components/Avatar/__tests__/Avatar.test.tsx#L5-L12)?